### ISSUE TITLE
Did you reproduce this article? I would like to ask how to train this model with my own training set

### ISSUE THREAD

**Issue by Fnothing** [myzhengSIMM/PBCNet#1]
I ran through the code of this article, but could only use the test set he provided, can this article be tested with its own dataset
